dark/index.html

```diff
@@ -151,7 +151,7 @@ <h1>
       <div id="tooltip">
         <span id="tooltiptext">Telegram</span>
         <a href="https://t.me/YoshithaRathnayake2008" target="_blank "><img
-            src="https://img.icons8.com/color/48/000000/telegram-app--v1.png" alt="Telegram Icon" /></a>
+            src="https://img.icons8.com/color/48/000000/telegram-app--v1" alt="Telegram Icon" /></a>

```

Comment:
`.png` is missing in the link @maddy020 
Add it.
```suggestion
            src="https://img.icons8.com/color/48/000000/telegram-app--v1.png" alt="Telegram Icon" /></a>
```